packages/neuron-ui/src/utils/formatters.ts

```diff
@@ -213,6 +213,58 @@ export const difficultyFormatter = (value: bigint) => {
   return `${localNumberFormatter(value)} H`
 }
 
+export const sudtAmountToValue = (amount: string = '0', decimal: string = '0') => {
```

Comment:
Can you add some tests for this function ?

Reply:
> Can you add some tests for this function ?

Tests are added